src/main/java/microsoft/exchange/webservices/data/EwsServiceXmlReader.java

```diff
@@ -62,8 +62,7 @@ private Date convertStringToDateTime(String dateTimeString) {
    * @throws java.text.ParseException
    */
   private Date convertStringToUnspecifiedDate(String dateTimeString) throws ParseException {
-    return this.getService().
-        convertStartDateToUnspecifiedDateTime(dateTimeString);
+    return getService().convertDateTimeStringToDate(dateTimeString, false);
```

Comment:
The convertDateTimeStringToDate method supports parsing time information, instead of date + timezone only. I don't know if this is desirable. When the server sends a time along with the date, this may cause issues instead of failing fast.